velox/common/memory/tests/MemoryAllocatorTest.cpp

```diff
@@ -30,6 +30,8 @@
 #include <gmock/gmock-matchers.h>
 #include <gtest/gtest.h>
 
+#include <fstream>
```

Comment:
Is this needed only for code inside #ifdef linux? If so, perhaps, wrap this with #ifdef as well.

Reply:
@mbasmanova Yes. Added `#ifdef linux`.